integration/cluster_test.go

```diff
@@ -485,14 +484,20 @@ func TestTransferLeaderStopTrigger(t *testing.T) {
 	defer clus.Terminate(t)
 
 	oldLeadIdx := clus.WaitLeader(t)
+	oldLeadID := clus.Members[oldLeadIdx].s.ID()
 	clus.Members[oldLeadIdx].StopWithAutoLeaderTransfer(t)
 
-	// issue put to one of the other member
-	kvc := toGRPC(clus.Client((oldLeadIdx + 1) % 3)).KV
-	sctx, scancel := context.WithTimeout(context.TODO(), clus.Members[oldLeadIdx].electionTimeout())
-	_, err := kvc.Range(sctx, &etcdserverpb.RangeRequest{Key: []byte("foo")})
-	scancel()
-	if err != nil {
-		t.Fatal(err)
+	// wait until two remaining members agree on the same leader
+	var remaining []*member
+	for i := range clus.Members {
+		if oldLeadIdx != i {
+			remaining = append(remaining, clus.Members[i])
+		}
+	}
+	clus.waitLeader(t, remaining)
```

Comment:
At least, I think the leader transition should complete and be reflected in the m.s.Lead() by this point. Or is there a delay between accepting the transfer and processing it into the leader update? In that case it should confirm the leader is either the new leader or transitions from the old leader to the new leader with no missing leader in between.